app/api/schema/access_codes.py

```diff
@@ -53,9 +53,13 @@ def validate_order_quantity(self, data, original_data):
                 data['tickets_number'] = access_code.tickets_number
 
         if 'min_quantity' in data and 'max_quantity' in data:
-            if data['min_quantity'] >= data['max_quantity']:
-                raise UnprocessableEntity({'pointer': '/data/attributes/min-quantity'},
-                                          "min-quantity should be less than max-quantity")
+            if data['min_quantity'] is not None and \
```

Comment:
Can ```'min_quantity' in data``` and ```data['min_quantity'] is not None``` be combined ?
Just asking. Not experienced in python

Reply:
Thought about it, we can do something like:
```
min_quantity = data.get('min_quantity', None)
max_quantity = data.get('max_quantity', None)
if min_quantity is not None and max_quantity is not None:
        if min_quantity >= max_quantity:
            raise UnprocessableEntity(
                {'pointer': '/data/attributes/min-quantity'},
                "min-quantity should be less than max-quantity"
            )
```

Reply:
Yes, looks a lot better.

Reply:
Should I do that, or wait for mentors' review?

Reply:
Let's wait
@iamareebjamal @poush @niranjan94 Please suggest if it's good.